pubnub-core/pubnub-core-api/src/main/kotlin/com/pubnub/api/retry/RetryConfiguration.kt

```diff
@@ -40,6 +40,11 @@ sealed class RetryConfiguration {
             excludedOperations: List<RetryableEndpointGroup> = emptyList(),
         ) : this(delayInSec.seconds, maxRetryNumber, excludedOperations, false)
 
+        // additional constructors for java
+        constructor() : this(MIN_DELAY, MAX_RETRIES_IN_LINEAR, emptyList())
```

Comment:
would `@JvmOverloads` on the secondary constructor work here? I think it will produce the same effect

Reply:
Unfortunately @JvmOverloads doesn't work for me here because it doesn't generate 
`constructor() : this(MIN_DELAY, MAX_RETRIES_IN_LINEAR, emptyList())`
and if I try to add it manually I get:
```
Platform declaration clash: The following declarations have the same JVM signature ( ()V): 
public constructor Linear() defined in com.pubnub.api.retry.RetryConfiguration.Linear
public constructor Linear(delayInSec: Int = ..., maxRetryNumber: Int = ..., excludedOperations: List<RetryableEndpointGroup> = ...) defined in com.pubnub.api.retry.RetryConfiguration.Linear
```

Reply:
OK, so I tested it and for some weird reason the IDE can't see the generated constructors, but they're there. We could release the library and users would see it. But let's leave your version for now to not have the error in the IDE. This must be an intelliJ/Kotlin bug, maybe worth reporting